maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt

```diff
@@ -168,6 +168,30 @@ class ClipboardPasteCommand : Command {
     }
 }
 
+class CopyTextCommand(
+    val selector: ElementSelector,
+): Command {
+
+    override fun description(): String {
+        return "Copy text from element with ${selector.description()}"
+    }
+
+    override fun injectEnv(env: Map<String, String>): CopyTextCommand {
+        return this
```

Comment:
You need to inject `env` into `selector` - this would allow users to use `env` parameters in a selector

Reply:
Can be done with `selector.injectEnv(env)`